yt_dlp/extractor/ard.py

```diff
@@ -601,3 +602,149 @@ def fetch_page(page_num):
         return self.playlist_result(
             OnDemandPagedList(fetch_page, self._PAGE_SIZE), full_id, display_id=display_id,
             title=page_data.get('title'), description=page_data.get('synopsis'))
+
+
+class ARDAudiothekIE(InfoExtractor):
+    _VALID_URL = r'''(?x)https:?//
+        (?:www\.)?ardaudiothek\.de/
+        (?:episode|(?P<playlist>sendung))/
+        (?:(?(playlist)[a-zA-Z-]+)/)?
+        (?P<id>urn:ard:(episode|section|extra|show):[a-f0-9]{16})/$'''
```

Comment:
I think this matchgroup is not relevant and can be ignored?  
```suggestion
        (?P<id>urn:ard:(?:episode|section|extra|show):[a-f0-9]{16})/$'''
```

Reply:
Thank you!